src/shared/onboarding/onboarding.css

```
@@ -0,0 +1,633 @@
+.tour-panel__target-unavailable strong,
+.tour-panel__target-unavailable span {
+  font-size: 12px;
+  line-height: 1.45;
+}
+
+.tour-panel__target-unavailable button {
+  min-height: 30px;
+  padding: 0 10px;
+  color: var(--color-warning-text);
+  background: transparent;
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)